index-fr.json

```diff
@@ -0,0 +1,737 @@
+"claque-merde": 2,
+"clille": 2,
+"comme papa dans maman": 2,
+"comme un chien fout sa merde": 2,
+"con": 2,
+"con comme un balai": 2,
+"con comme un gland": 2,
+"con comme une bite": 2,
+"conchier": 2,
+"connard": 2,
+"connarde": 2,
+"connasse": 2,
+"conneau": 2,
+"connerie": 2,
+"contrecrisser": 2,
+"copain de baise": 2,
+"copine de baise": 2,
+"coquillard": 2,
+"couille": 2,
+"couille de loup": 2,
+"couille molle": 2,
+"couilles au poteau": 2,
+"couillu": 2,
+"coup de pute": 2,
+"couter la peau des couilles": 2,
+"coûter la peau des couilles": 2,
+"couter la peau du cul": 2,
+"coûter la peau du cul": 2,
+"coûter une couille": 2,
+"cramouille": 2,
+"craquer son slip": 2,
+"craquer son string": 2,
+"cravacher le pur-sang": 2,
+"crisse": 2,
+"crissement": 2,
+"crosse": 2,
+"crosser": 2,
+"cruchasse": 2,
+"cul": 2,
+"culasse": 2,
+"culbuter": 2,
+"culer": 2,
+"culeter": 2,
+"culeur": 2,
+"dans ton cul": 2,
+"de mes couilles": 2,
+"de mes deux": 2,
+"de mon vier": 2,
+"débagouler": 2,
+"débander": 2,
+"déboiter": 2,
+"débroussailler la tranchée": 2,
+"décâlisser": 2,
+"décharger": 2,
+"déconnade": 2,
+"déconnage": 2,
+"déconner": 2,
+"décrisser": 2,
+"déculer": 2,
+"défoncer": 2,
+"défonceur": 2,
+"défourailler": 2,
+"dégager": 2,
+"déglinguer": 2,
+"dégueulassement": 2,
+"démerde yourself": 2,
+"demi-molle": 2,
+"démonter": 2,
+"dépucelage": 2,
+"dépuceler": 2,
+"dérouler du câble": 2,
+"des couilles": 2,
+"désengorger": 2,
+"diarrhée": 2,
+"Dieu me pignole": 2,
+"Dieu me tripote": 2,
+"dilater": 2,
+"djoc": 2,
+"DMC": 2,
+"does": 2,
+"doeser": 2,
+"doigter": 2,
+"drisser": 2,
+"drouille": 2,
+"DTC": 2,
+"ducon": 2,
+"duconnot": 2,
+"dugland": 2,
+"éburné": 2,
+"écouillé": 2,
+"emmanché": 2,
+"emmancher": 2,
+"emmerdé": 2,
+"emmerder": 2,
+"emmerdeur": 2,
+"empapaoutage": 2,
+"en avoir deux": 2,
+"en avoir plein le cul": 2,
+"en avoir plein les couilles": 2,
+"en avoir ras les burnes": 2,
+"en chier": 2,
+"enconnade": 2,
+"enconner": 2,
+"encuguler": 2,
+"enculable": 2,
+"enculade": 2,
+"enculade de mouche": 2,
+"enculagailler": 2,
+"enculage": 2,
+"enculage de mouche": 2,
+"enculage de mouches": 2,
+"enculailleur": 2,
+"enculatoire": 2,
+"enculé": 2,
+"enculé de ta mère": 2,
+"enculé de ta race": 2,
+"enculement": 2,
+"enculer": 2,
+"enculer les mouches": 2,
+"enculerie": 2,
+"enculette": 2,
+"enculeur": 2,
+"enculeur de mouches": 2,
+"enculeuse": 2,
+"enfant de chienne": 2,
+"enfant de putain": 2,
+"enfant de pute": 2,
+"enfiler": 2,
+"enfler": 2,
+"enfoiré": 2,
+"enfoirer": 2,
+"enfourailler": 2,
+"enfoutrer": 2,
+"englander": 2,
+"engrosser": 2,
+"engueuler": 2,
+"entre couilles": 2,
+"entuber": 2,
+"envoyer": 2,
+"envoyer chier": 2,
+"envulver": 2,
+"esque": 2,
+"estie": 2,
+"et mon cul, c’est du poulet": 2,
+"et ta connerie": 2,
+"étrangler le borgne": 2,
+"être chié": 2,
+"être couillu": 2,
+"être la fête": 2,
+"être mou de la bite": 2,
+"exploser": 2,
+"exploser le terrier": 2,
+"face de pet": 2,
+"faciale": 2,
+"faire": 2,
```

Comment:
As I said in my previous comment, I would give 'faire'  here a 0 rather than a 1 since this is literally just the verb "do" in french

Reply:
change done. Thanks a lot ! :)